create_blog_post_using_azure_openai_and_publish_in_medium/main.bal

```diff
@@ -0,0 +1,45 @@
+import ballerina/io;
+import ballerinax/azure.openai.text;
+import ballerinax/medium;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+configurable string mediumToken = ?;
+
+const API_VERSION = "2023-03-15-preview";
+
+public function main(string title) returns error? {
+    final medium:Client medium = check new ({auth: {token: mediumToken}});
+    medium:UserResponse userResponse = check medium->getUserDetail();
+    string? userId = userResponse.data?.id;
+
+    if userId is () {
+        return error("Medium user ID is not valid");
+    }
+
+    text:Deploymentid_completions_body completionsBody = {
+        prompt: string `Write a blog article on ${title} in markdown format.`,
+        max_tokens: 1000
+    };
+    final text:Client azureOpenAI = check new (
```

Comment:
```suggestion
    final text:Client azureOpenAIClient = check new (
```

Reply:
When we did the previous samples, @sameerajayasoma suggested that we define the client variables without the `Client` suffix to better represent the service.

Reply:
Ack. +1 to keep it as is